Marlin/src/lcd/extui/lib/mks_ui/draw_about.cpp

```diff
@@ -73,42 +74,46 @@ void lv_draw_about(void) {
 
   lv_refr_now(lv_refr_get_disp_refreshing());
 
-  LV_IMG_DECLARE(bmp_pic);
 
-  /*Create an Image button*/
+  // Create an Image button
   buttonBack = lv_imgbtn_create(scr, NULL);
 
   #if 1
-    lv_obj_set_event_cb_mks(buttonBack, event_handler, ID_A_RETURN, "bmp_return.bin", 0);
-    lv_imgbtn_set_src(buttonBack, LV_BTN_STATE_REL, &bmp_pic);
-    lv_imgbtn_set_src(buttonBack, LV_BTN_STATE_PR, &bmp_pic);
+    lv_obj_set_event_cb_mks(buttonBack, event_handler, ID_A_RETURN, NULL, 0);
+    lv_imgbtn_set_src(buttonBack, LV_BTN_STATE_REL, "F:/bmp_return.bin");
+    lv_imgbtn_set_src(buttonBack, LV_BTN_STATE_PR, "F:/bmp_return.bin");
     lv_imgbtn_set_style(buttonBack, LV_BTN_STATE_PR, &tft_style_label_pre);
     lv_imgbtn_set_style(buttonBack, LV_BTN_STATE_REL, &tft_style_label_rel);
+    #if HAS_ROTARY_ENCODER
+      if (gCfgItems.encoder_enable) lv_group_add_obj(g, buttonBack);
+    #endif
   #endif
 
   lv_obj_set_pos(buttonBack, BTN_X_PIXEL * 3 + INTERVAL_V * 4, BTN_Y_PIXEL + INTERVAL_H + titleHeight);
   lv_btn_set_layout(buttonBack, LV_LAYOUT_OFF);
-  /*Create a label on the Image button*/
 
+  // Create a label on the Image button
   label_Back = lv_label_create(buttonBack, NULL);
 
   if (gCfgItems.multiple_language != 0) {
     lv_label_set_text(label_Back, common_menu.text_back);
     lv_obj_align(label_Back, buttonBack, LV_ALIGN_IN_BOTTOM_MID, 0, BUTTON_TEXT_Y_OFFSET);
   }
 
-  fw_version = lv_label_create(scr, NULL);
-  lv_obj_set_style(fw_version, &tft_style_label_rel);
-  lv_label_set_text(fw_version, SHORT_BUILD_VERSION);
-  lv_obj_align(fw_version, NULL, LV_ALIGN_CENTER, 0, -60);
+  //fw_version = lv_label_create(scr, NULL);
+  //lv_obj_set_style(fw_version, &tft_style_label_rel);
+  //lv_label_set_text(fw_version, SHORT_BUILD_VERSION);
+  //lv_obj_align(fw_version, NULL, LV_ALIGN_CENTER, 0, -60);
 
   fw_type = lv_label_create(scr, NULL);
   lv_obj_set_style(fw_type, &tft_style_label_rel);
   lv_label_set_text(fw_type,
     #if MB(MKS_ROBIN_PRO)
-      "Firmware: Robin_Pro35"
+      "Firmware: Marlin-bugfix2.0.x-MKS-2.1.0"
```

Comment:
Won't this show `bugfix2.0.x-MKS-2.1.0` in *all* tagged releases now? Using something like `SHORT_BUILD_VERSION` would probably be ideal if you are wanting to list the Marlin version.

Reply:
... or was the intention of `fw_type` to list the motherboard? If that's the case, then using `MOTHERBOARD` would be a better choice. Your `#else` has `CUSTOM_MACHINE_NAME`, so it's not clear what you're trying to display here.

Reply:
I agree. It need to be fixed. The about must reflect the board/machine/version and not a hardcoded value.